projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -23,17 +23,17 @@ public partial class BirthRecord
         //
         /////////////////////////////////////////////////////////////////////////////////
 
-        /// <summary>Birth Record Identifier, Birth Certificate Number.</summary>
+        /// <summary>Birth Certificate Number.</summary>
         /// <value>a record identification string.</value>
         /// <example>
         /// <para>// Setter:</para>
         /// <para>ExampleBirthRecord.Identifier = "42";</para>
         /// <para>// Getter:</para>
         /// <para>Console.WriteLine($"Birth Certificate Number: {ExampleBirthRecord.Identifier}");</para>
         /// </example>
-        [Property("Identifier", Property.Types.String, "Birth Certification", "Birth Certificate Number.", true, IGURL.CertificateNumber, true, 3)]
+        [Property("Certificate Number", Property.Types.String, "Birth Certification", "Birth Certificate Number.", true, IGURL.CertificateNumber, true, 3)]
         [FHIRPath("Bundle", "identifier")]
-        public string Identifier
+        public string CertificateNumber
         {
             get
             {
```

Comment:
The ExtensionURL for CertificateNumber should come from the common library, it looks like this is pointing to the BFDR, see the BFDR ci build, you might need to generate the URLs for vital records if this one isn't defined in our library yet https://build.fhir.org/ig/HL7/fhir-bfdr/StructureDefinition-Bundle-document-birth-report.html